win_etw_macros/src/lib.rs

```diff
@@ -1104,17 +1140,31 @@ struct EventAttributes {
     level: syn::Expr,
     opcode: syn::Expr,
     task: syn::Expr,
+    event_id: Option<u16>,
     method_attrs: Vec<syn::Attribute>,
 }
 
+fn lit_to_string(lit: &Lit) -> String {
+    match lit {
+        Lit::Bool(b) => b.value().to_string(),
+        Lit::Byte(b) => b.value().to_string(),
+        Lit::ByteStr(b) => unsafe { String::from_utf8_unchecked(b.value()) },
```

Comment:
Please do not use unsafe code; it is not merited in a component like this. Use `String::from_utf8().unwrap()`, if you must.